www/include/configuration/configKnowledge/display-serviceTemplates.php

```diff
@@ -87,6 +73,10 @@
 $tpl = initSmartyTpl($modules_path, $tpl);
 
 try {
+    $postServiceTemplate = !empty($_POST['searchServiceTemplate'])
+    ? filter_input(INPUT_POST, 'searchServiceTemplate', FILTER_SANITIZE_STRING)
+    : '';
+
     $conf = getWikiConfig($pearDB);
     $WikiURL = $conf['kb_wiki_url'];
 
```

Comment:
did you chack if variables are user in the search.php script on line 83 ?
like : $_POST['searchTemplatesWithNoProcedure']